recipes/brev/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  name: brev
+  version: "0.6.310"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/brevdev/brev-cli/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: d7d8a58878c037372022f0c1d8fa58e15eb3d198498b521332d05def84cff351
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    - go-licenses save --save_path ../library_licenses .
```

Comment:
I'm a go reviewer, but shouldn't this create a license(s) file(s) ? If so, they must be listed in the license metadata.

Reply:
Good catch, missed that